providers/pool.rb

```diff
@@ -221,19 +248,17 @@ def configure
       cmd << " \"/[name='#{@new_resource.pool_name}'].processModel.password:#{@new_resource.pool_password}\""
       Chef::Log.debug(cmd)
```

Comment:
I think the above was_updated came from that upstream merge.  Should be @was_updated?